components/Chat.js

```diff
@@ -3,8 +3,9 @@ import { StyleSheet, View, TouchableOpacity, SafeAreaView, KeyboardAvoidingView,
 import { GiftedChat, Send, InputToolbar, Composer, Bubble, Time } from 'react-native-gifted-chat'
 import { PreferencesContext } from './PreferencesContext';
 import {
-    Avatar, useTheme, Appbar, IconButton, Card, Title, Paragraph, List, Text, Button, TouchableRipple, Switch, TextInput, Searchbar
+    Avatar, useTheme, Appbar, IconButton, ToggleButton, SegmentedButtons, Card, Title, Paragraph, List, Text, Button, TouchableRipple, Switch, TextInput, Searchbar
 } from 'react-native-paper';
+import uuid from 'react-native-uuid';
 import { robotBase64 } from '../assets/logo';
```

Comment:
Not sure for the reason to include data:image/jpeg;base64, ; 

I will follow your choice anyway. 

So, any base64 string stored should have this leading string